models/ed/R/write.configs.ed.R

```diff
@@ -181,42 +177,80 @@ write.config.ED2 <- function(defaults, trait.values, settings, run.id){
       ##
       
     } else {
-      ##is a PFT
-      pft <- defaults[[group]]
-      ## Insert PFT constants into output xml file  
-      pft.xml <- listToXml(pft$constants, 'pft')
-      ## Insert PFT names into output xml file
-      pft.xml <- append.xmlNode(pft.xml, xmlNode("name", pft$name))
-      
-      ##TODO this should come from the database
-      edpft <- pftmapping$ED[which(pftmapping==group)]
-      if (is.null(edpft)) {
-        logger.warn("No mapping found for", group, "using 1")
-        edpft <- 1
-      }
-      
+          ##is a PFT
+#       pft <- defaults[[group]]
+#       ## Insert PFT constants into output xml file  
```

Comment:
Sorry, yep... had that hanging around to try to make sure I wasn't dropping any functionality. Deleted in latest commit.